<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventaire</class>
 <widget class="QMainWindow" name="Inventaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inventaire</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Champ de Recherche</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lblSearchBox"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableStock">
      <property name="locale">
       <locale language="French" country="Switzerland"/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Code article</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Marque</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mod√®le</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Taille</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stock</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
